src/tattoo.ts

```diff
@@ -366,6 +423,12 @@ async function _main(elements: ElementRepo[]) {
      fs.mkdirSync("repos");
   }
 
+  let configFile = opts["branchconfig"];
```

Comment:
future: you might want to checkout liftoff. It's what gulp uses to find config files, and it'll search a path. This might be useful if we think a user might go into the repos and edit files to figure out a bug. Then they could run tattoo from any subdirectory.